File: app/src/main/java/com/github/willjgriff/ethereumwallet/ui/settings/mvp/ChangeAddressView.java
package com.github.willjgriff.ethereumwallet.ui.settings.mvp;

import com.github.willjgriff.ethereumwallet.ethereum.address.model.DomainAddress;

import java.util.List;

/**
 * Created by Will on 28/02/2017.
 */

public interface ChangeAddressView {

	void setAddresses(List<DomainAddress> allAccounts);

	void closeScreen();
}

File: app/src/main/kotlin/com/github/willjgriff/ethereumwallet/ui/settings/mvp/ChangeAddressView.kt
package com.github.willjgriff.ethereumwallet.ui.settings.mvp

import com.github.willjgriff.ethereumwallet.ethereum.address.model.DomainAddress

/**
 * Created by williamgriffiths on 04/05/2017.
 */
interface ChangeAddressView {

    fun setAddresses(allAccounts: List<DomainAddress>)

    fun closeScreen()
}